src/field/arena.ts

```diff
@@ -469,10 +567,22 @@ export class Arena {
     }
   }
 
+  /**
+   * Gets the time of day
+   * | time of day | waveCycle | waves |
+   * |-------------|-----------|-------|
+   * | day         | 0-14      | 15    |
+   * | dusk        | 15-19     | 5     |
+   * | night       | 20-34     | 15    |
+   * | dawn        | 35-39     | 5     |
+   *
```

Comment:
```suggestion
   * ```
   * | time of day | waveCycle | waves |
   * |-------------|-----------|-------|
   * | day         | 0-14      | 15    |
   * | dusk        | 15-19     | 5     |
   * | night       | 20-34     | 15    |
   * | dawn        | 35-39     | 5     |
   * ```
```